unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/Channels/ChatChannelsControllerMock.cs

```diff
@@ -293,5 +327,64 @@ private async UniTask MuteFakeChannel(string channelId)
             };
             controller.UpdateChannelInfo(JsonUtility.ToJson(msg));
         }
+
+        private async UniTask SimulateDelayedResponseFor_ChatInitialization()
+        {
+            await UniTask.Delay(Random.Range(50, 1000));
+
+            var payload = new InitializeChatPayload
+            {
+                totalUnseenPrivateMessages = Random.Range(0, 5)
+            };
+            controller.InitializeChat(JsonUtility.ToJson(payload));
+        }
+
+        private async UniTask SimulateDelayedResponseFor_ChannelsInitialization()
+        {
+            await UniTask.Delay(Random.Range(50, 1000));
+
+            var payload = new InitializeChannelsPayload
+            {
+                unseenTotalMessages = Random.Range(0, 5)
+            };
+            controller.InitializeChannels(JsonUtility.ToJson(payload));
```

Comment:
why we need a separated message for channel initialization? i think kernel should calculate all unseen messages at `InitializeChat` message

Reply:
We defined this as 2 separated initializations in our RFCs:
![image](https://user-images.githubusercontent.com/64659061/180414599-bb781834-04ce-4285-8a73-64f556944461.png)
![image](https://user-images.githubusercontent.com/64659061/180414659-c83627a5-e578-4f6f-8148-21eaadb4b3a3.png)